ntbs-service/Helpers/ValidationExtensions.cs

```diff
@@ -0,0 +1,52 @@
+using System;

```

Comment:
This is an alternative approach (not finished) to having a validation service - we could create extension models for page model (so this class should really be called PageModelExtensions) for all the validation methods. They would then be called like
`this.ValidateProperty(...)` rather than `validationService.ValidateProperty)...)`

Do you think that would be preferable?

Reply:
I completely agree with you here - just wanted to see whether you had any other pros for extension methods that might make them preferable. It does make more sense for me for this to be a service, so I'll stick with that and remove the extension :)

Reply:
yeah, i guess if we wanted to be pedantic about it we could call it a "factory" and have a method on it that creates the thing for us... but that would be formality for formality's sake at this point. Let's just use `new` (and not tell anyone about it ;) 🙊)